controller/controller.js

```diff
@@ -144,6 +168,19 @@ module.exports = class Controller extends BaseController {
   _getErrors(req, res, callback) {
     super._getErrors(req, res, () => {
       Object.keys(req.form.errors).forEach(key => {
+        var field = req.form.options.fields[key];
+        // get first option for radios
+        if(field.mixin == 'radio-group') {
+          req.form.errors[key].errorLinkId = key + "-" + field.options[0];
+        }
+        // get first field for date input control
+        else if (field && field.controlType === 'date-input') {
+          req.form.errors[key].errorLinkId = key + '-day';
+        }
+        else {
+          req.form.errors[key].errorLinkId = key;
+        }
```

Comment:
nhs found that they could not navigate to the error on a page particular for dob.  This is why they have made this changes